TarponSimulator2017/View/Player.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework.Input;
+
+using Core;
+
+namespace TarponSimulator2017
+{
+	public class Player : Sprite
+	{
+		public Player ()
+		{
+		}
+
+
+
+		protected Boat boat;
+        protected Texture2D[] _tab_chiffre;
```

Comment:
Small details, but:

  * You prefix your private variables by a tabulation. That's not necessary in C#
  * The indentation is not correct